docs/organizations/managing-security-and-risk.md

```diff
@@ -534,12 +534,52 @@ You're also able to click any dependency to find out more information about it.
\ No newline at end of file
+<sup><span id="license-scanning">6</span></sup>: Visit the [supported languages and tools](../getting-started/supported-languages-and-tools.md#supported-languages-and-tools) page for a list of supported languages.  
+
+
+## App scanning {: id="app-scanning"}
+
+!!! important
+    App scanning is a business feature. If you are a Codacy Pro customer, contact our customer success team to access a short trial.
+
+The **Security and risk management app scanning** page allows organizations to scan APIs and Web Applications for security vulnerabilities. This is part of our DAST (Dynamic Application Security Testing) capabilities, powered by ZAP.
+
+To access the app scanning page, access the [overview page](#dashboard) and click the **App scanning** tab.
+
+![Security and risk management app scanning page](images/security-risk-management-app-scanning.png)
+
+App scanning tests applications in real-world scenarios, making it possible to find configuration and authentication issues or other runtime vulnerabilities that may impact your application’s functionality and security. It’s also a good method for preventing regressions and doesn’t depend on a specific programming language. As long as the application can be accessed through a browser, a DAST tool can typically scan it for vulnerabilities.
+
+!!! note
+    Already using ZAP? [Upload your results via the API.](../codacy-api/examples/uploading-dast-results.md)
+
+### How to scan a target
+To scan a target, you can either go to the Security and Risk Management dashboard and access the App Scanning tab, or set it up for automation using our API.
+
+!!! important
+    Only admins and organization managers will be able to create new targets and run scans (both in-app and via the API).
+
+
+<div>
+  <iframe width="100%" height="472" src="https://www.youtube.com/embed/qPwHlIGJYXs?si=Q2Y5cmRelodtc05n" title="DAST targets" frameborder="0" 
+allowfullscreen>
+  </iframe>
+</div>
+
+
+From within the tab, you're able to configure a new target by inputting the URL of the app or API you'd like to scan. You can configure up to 6 targets within your organization (if you need more, talk to your customer success representative).
+Scans can be triggered manually via the app or the API. As you click to start a scan, it will be first added to that target's queue and then run. If you've got a new version of the app while running a scan, no problem: just add a new scan to the queue, and it will run immediately after. There are no limits to how many scans an organization can run per target, so this should accommodate all your deployment needs.
+Once a scan is complete, its findings will be added to the findings tab, where you can review them using the filter **Scan types > DAST/App scanning**.
+
+!!! note
+    Currently, DAST issues are only visible to admin and organization admin roles. We'll be reviewing this issue soon.
```

Comment:
```suggestion
!!! note
    Currently, DAST issues are only visible to admin and organization admin roles.
```
I would not make this sort of comments. We are planning to address this soon but who knows when it actually gets done.